iogt/static/css/modules/_m-navigation.scss

```diff
@@ -19,7 +19,7 @@
  
 .menu-list__item--title {
   display: block; 
-  font-size: 14px;
+  font-size: $font-middle;
```

Comment:
`medium` may be a better word, based off of standard CSS sizes:

font-size: xx-small;
font-size: x-small;
font-size: small;
font-size: medium;
font-size: large;
font-size: x-large;
font-size: xx-large;

Is medium a good size here? Since it's smaller than the standard of 16px?